parts/k8s/kubernetescustomscript.sh

```diff
@@ -57,20 +57,15 @@ else
     FULL_INSTALL_REQUIRED=true
 fi
 
-holdWALinuxAgent() {
-    if [[ $OS == $UBUNTU_OS_NAME ]]; then
-        wait_for_apt_locks
-        retrycmd_if_failure 120 5 25 apt-mark hold walinuxagent || exit $ERR_HOLD_WALINUXAGENT
-        wait_for_apt_locks
-    fi
-}
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
```

Comment:
do we nee to check for Ubuntu OS here? Isn't the shellscript only for linux/ubuntu?

Reply:
that check was there before, I believe it might be for CoreOS. In any case, if it can be removed let's not remove it in a hotfix PR. Let's make minimal changes in here and fix this in another PR.